explorer/explorer.go

```diff
@@ -318,6 +319,13 @@ func New(cfg *ExplorerConfig) *explorerUI {
 	exp.NetName = netName(exp.ChainParams)
 	exp.MeanVotingBlocks = txhelpers.CalcMeanVotingBlocks(params)
 
+	var premine int64
+	ledger := params.BlockOneLedger
+	for i := range ledger {
+		premine += ledger[i].Amount
```

Comment:
While this is spot on, there's actually func `(p *Params) BlockOneSubsidy() int64`, which we might as well use:

https://github.com/decred/dcrd/blob/e69a789183f39c2991a0da78c0b056887abe1d36/chaincfg/params.go#L664-L677

Reply:
Oh nice.